source/rst/tax_smoothing_1.rst

```diff
@@ -19,12 +19,12 @@ How to Pay for a War: Part 1
 ============================
 
 An Application of Markov Jump Linear Quadratic Dynamic Programming
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------------------------------------------------------
 
 
 This notebook constructs generalizations of Barro’s classic 1979 model
 of tax smoothing. Our generalizations are adaptations of extensions of
-his 1979 model suggested by Barro (1999, 2003)
+his 1979 model suggested by Barro (1999, 2003) :cite:`Barro1979`
```

Comment:
this is the wrong year citation -- you need to look for papers from 1999 and 2003